### DISCUSSION TITLE
Unable to run exclusive fullscreen mode on lazer

### DISCUSSION THREAD

**Discussion by thdabezt** [ppy/osu#20232] [Q&A] [answered]
### Type

Game behaviour

### Bug description

My osu!lazer doesn't run exclusive fullscreen mode while stable can. I checked if this is a bug with lazer not detecting exclusive fullscreen but I can open game bar in osu!lazer, in stable it just tabbed out.
I'm playing on my laptop.
- OS : Windows 10
- CPU : 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz   2.30 GHz
- GPU : NVIDA GeForce RTX 3070 Laptop GPU

### Screenshots or videos

![Capture](https://user-images.githubusercontent.com/52331752/189515111-2cb119f4-fc72-41db-87b5-0e3ab4e33ac4.PNG)

### Version

2022.910.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/9541900/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9541901/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9541902/updater.log)
[database.log](https://github.com/ppy/osu/files/9541903/database.log)

**Comment by peppy** [comment, marked as answer]
Stable is probably running in compatibility (directx) mode due to an optimus gpu being detected.

We're working on adding similar functionality to lazer, should arrive in the next month or three.

**Reply by Nauj-Montes** [reply]
😭😭😭

**Comment by ItsukaHiro** [comment]
Same issue here, and I didn't find out anyway to run in exclusice fullscreen

**Reply by peppy** [reply]
Please see https://github.com/ppy/osu/discussions/20232#discussioncomment-3617333.

**Comment by stinkzies** [comment]
The same happens with integrated graphics (mine is Intel HD 530)

**Reply by peppy** [reply]
If you're running an intel card, you can ignore the warning. It will disappear in a future release as it is a false positive.

**Reply by mirh** [reply]
Any update?

**Reply by peppy** [reply]
Wait for the new renderer which will fix this.

**Reply by Yandall** [reply]
Still no update? I really want to play osu lazer

**Reply by Tran-Antoine** [reply]
Used to have this issue. The new renderer update completely solves it for me.

**Reply by Nauj-Montes** [reply]
> Still no update? I really want to play osu lazer

Indeed my friend, this issue was resolved long ago and it was due to some intel drivers.

**Reply by London-2** [reply]
still not working and im on intel >_<

**Reply by mirh** [reply]
Did you update your drivers?

**Reply by London-2** [reply]
i did
